install-config.yaml

```diff
@@ -4,33 +4,26 @@ apiVersion: v1
 baseDomain: testing
 compute:
 - architecture: 
-  hyperthreading: Enabled
   name: worker
-  platform: {}
-  replicas: 1
+  replicas: 0
 controlPlane:
   architecture: 
-  hyperthreading: Enabled
   name: master
-  platform: {}
   replicas: 1
 metadata:
-  creationTimestamp: null
   name: crc
 networking:
   clusterNetwork:
   - cidr: 10.217.0.0/22
     hostPrefix: 23
   machineNetwork:
   - cidr: 192.168.126.0/24
-  networkType: OpenShiftSDN
```

Comment:
I assume the default networkType will be OVNKubernetes? Were there any measurements made with the new bundles ? https://github.com/crc-org/snc/pull/851#issuecomment-1942017621

Reply:
We need to remove the OVNKubernetes mentions from the PR/logs/..., or reword it :-/

Reply:
I think I removed it from the commit message will remove it from this PR description also.

Reply:
Ah ok  good then, I haven't double checked